### ISSUE TITLE
Conformance critieria for headers with multiple field values

### ISSUE THREAD

**Issue by kjetilk** [solid/solid-spec#194]
When writing a test for [`OPTIONS`](https://github.com/solid/solid-spec/blob/b941ff795acdedb7d7a24d40dabdfcce7efa9283/api-rest.md#discovering-server-capabilities---the-options-method), it is not clear what the conformance criteria would be. In particular, how  the multiple field values should be treated, would an implementation be conformant if it had a subset of the indicated values? Exactly the same? Superset?

**Comment by RubenVerborgh** [comment]
That and CORS, I would say.

**Comment by kjetilk** [comment]
Indeed. There's a bunch of them. And it is an open world in that RFC7230 does not define the syntax of field values, but I think a comma separated list will work for now.
